src/services/groups/groupService.spec.ts

```diff
@@ -0,0 +1,98 @@
+import "regenerator-runtime/runtime";
+import faker from "faker";
+
+import sequelize from "../../data-acess/db";
+import { Group as GroupModel } from "../../models/Group";
+import { Group as GroupType, GroupInternalProps } from "../../types/group";
+import GroupService from "./groupService";
+
+const MOCK_UUID = '5543beb0-716f-4405-ade9-2519097a88c2';
+
+describe('Groups service', () => {
+    const service = new GroupService();
+
+    let mockGroupId: string;
+    const defaultGroupFields: Omit<GroupType, GroupInternalProps> = {
+        name: faker.company.companyName(),
+        permissions: ['SHARE', 'DELETE', 'UPLOAD_FILES'],
+    };
+    const _createGroup = async (groupFields?: Partial<GroupType>) => {
+        return await GroupModel.create({
+            id: mockGroupId,
+            ...defaultGroupFields,
+            ...groupFields,
+        });
+    };
+
+    beforeAll(async () => {
+        await sequelize.sync();
```

Comment:
Your test suits should not communicate with the db. Firstly, it may accidentally corrupt the data. Secondly, your tests will be slower. So, it's better to avoid the usage of real sequalize and use the mocked one instead